packages/nx/src/project-graph/project-graph-builder.spec.ts

```diff
@@ -91,13 +93,6 @@ describe('ProjectGraphBuilder', () => {
         'target'
       )
     ).toThrowError();
-    expect(() =>
```

Comment:
This no longer breaks, but does not create dependency in `dependencies` either, only in `files`